bot/index.js

```diff
@@ -1,156 +1,161 @@
-const Logger = require("./utils/logger.js");
-Logger.setSeverity("info");
+async function init() {
```

Comment:
I turned the index.js file to async, because it had some .then chains which I did not want to continue for maintainability reasons.

Reply:
Opened #90